requirements.json

```diff
@@ -177,5 +177,30 @@
         "sha256sum": "d65a1d1a050e4fd7912c18468954f64db824a1e1b621235a153d010beae14757"
       }
     }
+  },
+  "kompose": {
+    "name": "Kompose",
+    "description": "Kompose takes a Docker Compose file and translates it into Kubernetes or OpenShift resources",
```

Comment:
That sounds better! `A tool to translate a Docker Compose file into Kubernetes or OpenShift resources` (adding the `a`)